@navikt/core/react/src/util/i18n/locales/nb.ts

```diff
@@ -44,4 +44,7 @@ export default {
       labelSuffix: "slett",
     },
   },
+  ErrorSummary: {
+    Heading: "Du må rette disse feilene før du kan fortsette:",
```

Comment:
We use UpperCamelCase for the component name, and lowerCamelCase for the message keys.
```suggestion
    heading: "Du må rette disse feilene før du kan fortsette:",
```

Reply:
av en grunn fikk jeg typescript issues med det (ikke her, men når jeg skulle bruke`Heading` i koden 😵 ). kan se å få det til igjen.

Reply:
Hva slags TS issues?